### ISSUE TITLE
Public Repo Work

### ISSUE THREAD

**Issue by libdgg** [emory-libraries/avalon#83]
Public repo
Name: avalon-terraform

NOTE: keep as a fork of avalon media system's avalon-terraform - allows us to bring in imports from their repo into ours. there is no contract for our avalon terraform to do what their avalon terraform does. our fork would not merge back into the main repo. 
How would Phuong work in our fork? Phuong can either use a personal fork or we can add hims as a contributor to our fork.
We manage our differences by setting up our new branch for Emory Main. Consistent approach to management. Default branches are called "main-emory". This means avalon can update theirs and it won't collide.

- [x] invite Phuong as contributor
- [x] setup a Main-Emory branch. (take what we have on master and rename, get a master branch that is up-to-date and FF to include all changes - the changes Solomon has in there will not be lost).
- [x] setup Main-Emory as default branch for the repo.

**Comment by rotated8** [comment]
I got `master` fast-forwarded to avalonmediasystem/avalon-terraform's `master`, but it took a little work. Here's what I did:

1. Add avalonmediasystem/avalon-terraform as a new remote (I called it "upstream"): `git remote add upstream git@github.com:avalonmediasystem/avalon-terraform.git`
2. Pull in info from that remote: `git fetch upstream`
3. Ensure you are on our master branch: `git checkout master`
4. Track the `master` branch from avalonmediasystem as the upstream branch of our `master` branch: `git branch -u upstream/master`

Now when changes are made in avalonmediasystem/avalon-terraform's `master`, we can pull them in and push them to GitHub's `master` with three steps:

Ensure you are on our master branch: `git checkout master`
Pull in changes from avalonmediasystem: `git pull`
Push changes to GitHub (assuming your GitHub remote is called "origin", as it is by default): `git push origin`
